ui/src/test/java/be/appwise/ui/ExampleUnitTest.kt

```diff
@@ -0,0 +1,17 @@
+package be.appwise.ui
+
+import org.junit.Test
+
+import org.junit.Assert.*
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
```

Comment:
Maybe we can just remove this ExampleUnitTest which is pregenerated as it has no use to us